### DISCUSSION TITLE
Community Resources

### DISCUSSION THREAD

**Discussion by j-p-courneya** [Relay-Maryland/relaymaryland.com#11] [Ideas]
### Historic Tax Credit Information
If you own a historic home in Relay/St. Denis, you may be able to get a tax credit for certain rehabilitation projects through the Baltimore County Historic Rehabilitation Tax Credit program. For information, including how to apply for the credit, visit the [Historic Rehabilitation Tax Credit page](https://www.baltimorecountymd.gov/departments/planning/historic_preservation/taxcreditfaq.html) on the Baltimore County Government website or click the link below.
[Baltimore County Historic Tax Credit](https://www.baltimorecountymd.gov/departments/planning/historic_preservation/taxcreditfaq.html)
### Preservation-Related Links
The Landmarks Preservation Commission, which reviews and approves plans for alterations to historic structures.
Maryland Inventory of Historic Properties (MIHP), which includes information often used to determine if a property is eligible for designation as a Baltimore County Landmark.
Maps of Baltimore County historic districts and more.
[Historic Preservation in Baltimore County](https://www.baltimorecountymd.gov/departments/planning/historic_preservation/index.html)
The PABC is a county-wide 501(c)3 organization dedicated to historic preservation in Baltimore County.
[Preservation Alliance of Baltimore County (PABC)](https://preservationabc.org/)
Preservation Maryland is a non-profit focused on revitalizing and reinvesting in historic communities, advocating, and building the historic trades workforce. [Preservation Maryland](https://www.preservationmaryland.org/)

### Maryland Historical Trust
The [Maryland Historical Trust](https://mht.maryland.gov/) is the state agency dedicated to preserving and interpreting the legacy of Maryland’s past – through research, conservation and education. Includes information on the state income tax credit for qualified rehabilitation of historic properties in Maryland. 
### National Register of Historic Places
The [National Register of Historic Places](https://www.nps.gov/subjects/nationalregister/index.htm) is the official list of the Nation’s historic places worthy of preservation. The website includes a helpful FAQ.  

### Security and Safety Contact numbers

1. **In Case of Emergency**, or if you see a crime in progress, call BR Police Dispatch at 410-887-0872, or call 911. If a Garden District security patrol officer happens to already be in the area, Dispatch will call them.

2. **Security Patrol**: For ongoing security concerns regarding criminal activity in the neighborhood, you can call the police dispatch phone number at 410-887-0872. PLEASE always start your message with your name, address, and phone number.

3. **To discuss ongoing security concerns** regarding criminal activity, please fill out a contact form here and report your concern. Our board member in charge of security will reach out to you regarding your question.

4. **House Vacancies** If you would like to inform the GD Security Patrol that you’ll be out of town, you can submit a house [vacancy notice](https://resources.baltimorecountymd.gov/Documents/Police/2014pdfs/vacanthouseform141119.pdf) or email it to your [precinct](https://www.baltimorecountymd.gov/departments/police/precinctsgeneral/precinctsall.html).

### 5 Steps to a Safer Neighborhood

![](https://i.imgur.com/9hN8fue.jpg)

### Home Security Resources and Links

These home security links and resources will help you prepare a secure home and help you know what to do in the event of a problem. Please download and print for your records.

* Website for Baltimore County Police Department [Precinct 1 – Wilkens](https://www.baltimorecountymd.gov/departments/police/pc01/index.html)

* Baltimore County Police Department Crime prevention tips for Homeowners [link](https://www.baltimorecountymd.gov/departments/police/yoursafety/crimeprev_homes.html)

* Residential Security Surveys can be requested by reaching the Precinct 1 – Wilkens Outreach Unit at : **410-887-5165**

* Tips for non-emergency situations can also be submitted online via the department's [iWatch Baltimore County tip form](https://www.baltimorecountymd.gov/departments/police/media/iWatch/index.html).

@brianzelip

**Comment by brianzelip** [comment]
Done
